### DISCUSSION TITLE
Fabric and Jan.AI

### DISCUSSION THREAD

**Discussion by cmidd1** [danielmiessler/Fabric#551] [Q&A]
I'm kindly asking for guidance with local private LLM usage. I installed fabric on my home linux pc, and used the --setup option, added my OpenAI key, I can listmodels and see gpt models.
I have jan.ai (is compatible with OpenAI API) on my home windows pc, running as a server with llama3-8b-instruct from Meta.
The linux web browser comms are good, URL (e.g. http://10.0.1.12:1337/) renders with an interface to models, chat, messages, threads, and assistants.
In my linux shell, I need to understand how to use the fabric environment variables and/or the option --remoteOllamaServer but so far I get Connection Error or a list of GPT models but not my local llama3? Hoping to see llama3 listed, and pass a simple question to the model getting an ai pattern output.
Thanks,
Chris

**Comment by cmidd1** [comment]
ohh silly me, env var was https but server is http, all OK now!

**Reply by fahadshery** [reply]
could you share your .env file please? I have ollama running locally. I want to use it by default when installing fabric

**Reply by xssdoctor** [reply]
If you want to run it by default, just change your default model to your Ollama model
